src/services/codefixes/fixAddMissingMember.ts

```diff
@@ -93,23 +110,56 @@ namespace ts.codefix {
         readonly declSourceFile: SourceFile;
         readonly isJSFile: boolean;
     }
-    type Info = EnumInfo | ClassOrInterfaceInfo;
 
-    function getInfo(tokenSourceFile: SourceFile, tokenPos: number, checker: TypeChecker, program: Program): Info | undefined {
+    interface FunctionInfo {
+        readonly kind: InfoKind.Function;
+        readonly call: CallExpression;
+        readonly token: Identifier;
+        readonly sourceFile: SourceFile;
+        readonly modifierFlags: ModifierFlags;
+        readonly parentDeclaration: SourceFile | ModuleDeclaration;
+    }
+
+    function getInfo(sourceFile: SourceFile, tokenPos: number, checker: TypeChecker, program: Program): Info | undefined {
         // The identifier of the missing property. eg:
         // this.missing = 1;
         //      ^^^^^^^
-        const token = getTokenAtPosition(tokenSourceFile, tokenPos);
+        const token = getTokenAtPosition(sourceFile, tokenPos);
         if (!isIdentifier(token) && !isPrivateIdentifier(token)) {
             return undefined;
         }
 
         const { parent } = token;
-        if (!isPropertyAccessExpression(parent)) return undefined;
+        if (isIdentifier(token) && isCallExpression(parent)) {
+            return { kind: InfoKind.Function, token, call: parent, sourceFile, modifierFlags: ModifierFlags.None, parentDeclaration: sourceFile };
+        }
+
+        if (!isPropertyAccessExpression(parent)) {
+            return undefined;
+        }
 
         const leftExpressionType = skipConstraint(checker.getTypeAtLocation(parent.expression));
         const { symbol } = leftExpressionType;
-        if (!symbol || !symbol.declarations) return undefined;
+        if (!symbol || !symbol.declarations) {
+            return undefined;
+        }
+
+        if (isIdentifier(token) && isCallExpression(parent.parent)) {
+            const moduleDeclaration = find(symbol.declarations, isModuleDeclaration);
+            const moduleDeclarationSourceFile = moduleDeclaration?.getSourceFile();
+            if (moduleDeclaration && moduleDeclarationSourceFile && !program.isSourceFileFromExternalLibrary(moduleDeclarationSourceFile)) {
+                return { kind: InfoKind.Function, token, call: parent.parent, sourceFile, modifierFlags: ModifierFlags.Export, parentDeclaration: moduleDeclaration };
+            }
+
+            const moduleSourceFile = find(symbol.declarations, isSourceFile);
+            if (sourceFile.commonJsModuleIndicator) {

```

Comment:
Why doesn't it work for common JS modules?

Reply:
@jessetrinity The issue description contains only samples with modules, https://github.com/microsoft/TypeScript/issues/26217, I decided that this is a limitation for common js. I think just need to check if `importAdder` supports common JS modules.